redis.conf

```diff
@@ -1762,3 +1762,29 @@ rdb-save-incremental-fsync yes
 # Maximum number of set/hash/zset/list fields that will be processed from
 # the main dictionary scan
 # active-defrag-max-scan-fields 1000
+
+############################### PMEM CONFIG ###############################
+
```

Comment:
Memory Allocation Policies allows modifying mechanism how heap memory is allocated via zmalloc()
function calls. It can target DRAM, Persistent Memory or both. In general, bigger allocations
should be stored in Persistent Memory which provides a higher capacity, while smaller and more
frequently used should be stored in DRAM.

When allocations are to be mixed on both types of memory, this can be defined with:
   threshold  – allocations smaller than the threshold will target DRAM, equal and bigger will target PMEM
   ratio      – the application will be checking DRAM and PMEM utilization ratio and will adopt internal 
                threshold value to reach the defined ratio